### PR TITLE
switch to mimic cpp implementation

### PR DESCRIPTION
**Pull request by benmaidel**
depends on https://github.com/ipa320/cob_driver/pull/344

### CODE REVIEW

**Review by floweisshardt — APPROVED** [review]
tested locally: OK

**Comment by floweisshardt** [comment]
why is ```cwd="node"``` neccessary?

File: cob_bringup/drivers/mimic.launch [hunk 1]
```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0"?>
 <launch>
-
-	<arg name="rotation" default="0"/>
-
-	<!-- start mimic -->
-	<node pkg="cob_mimic" type="mimic_node.py" name="mimic" cwd="node" respawn="false">
-		<param name="rotation" value="$(arg rotation)"/>
```
Comment (fmessmer) [inline cob_bringup/drivers/mimic.launch:8]: @ipa-bnm @ipa-fmw @ipa-nhg
just to confirm....there is no need for `rotation` argument on the robots anymore?
I remember @ipa-nhg mentioning that some head displays are configured different than other...on which one did you test?

simulation is fine...

Comment (floweisshardt) [reply cob_bringup/drivers/mimic.launch:8]: there's no need for the rotation any more, we'll rotate the ubuntu system settings

**Pull request merged**